### PR TITLE
Erik Golden addition to december 2017 attendance

### PR DESCRIPTION
**Pull request by erikgolden7**

### CODE REVIEW

**Comment by m2mathew** [comment]
@erikgolden7 Let's leave the single quotes in place (except in JSX). That's how I usually roll...because of my normal linting rules. 😄

**Comment by erikgolden7** [comment]
Sorry about the doubles, you can thank prettier for that!

File: src/shared/december2017data.js [hunk 1]
```diff
@@ -24,33 +25,37 @@ const december2017data = [
...
     name: 'Dallas Stars',
     twitter: 'DallasStars',
-    about: 'Hat tricks a plenty',
+    about: 'Hat tricks a plenty'
   },
   {
     name: 'Dallas Cowboys',
     twitter: 'dallascowboys',
-    about: 'America\'s team for American-style football',
+    about: 'America's team for American-style football'
```
Comment (m2mathew) [inline src/shared/december2017data.js]: Be careful to escape a single quote (or double quote) when inside other single (double) quotes. That's actually a good argument to use double quotes everywhere. 🤔

File: src/shared/december2017data.js [hunk 2]
```diff
@@ -4,12 +4,13 @@ const december2017data = [
   {
     name: 'Mike Mathew',
     twitter: 'drumsensei',
-    about: 'Father of three, husband of one, writer of JavaScripts, drinker of coffee',
+    about:
```
Comment (m2mathew) [inline src/shared/december2017data.js]: This can stay on a single line, yeah?

Comment (erikgolden7) [reply src/shared/december2017data.js]: Yep, prettier again... it cuts it down at a certain width

File: src/shared/december2017data.js (hunk 1 above)
Comment (erikgolden7) [reply src/shared/december2017data.js]: That's the reason I chose doubles. There is no wrong, just different! 😄

**Comment by m2mathew** [comment]
ok, ok...I really will merge yours next this time! This is more of that "real world" app type stuff with so many conflicts in a single file. 😜

**Comment by m2mathew** [comment]
👍 I will overlook the semicolon and trailing commas since we have no linter running. 😆

**Pull request merged**